pkgs/by-name/te/terragrunt-atlantis-config/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildGo125Module,
+  fetchFromGitHub,
+  versionCheckHook,
+  mockgen,
+}:
+buildGo125Module (finalAttrs: {
+  pname = "terragrunt-atlantis-config";
+  version = "1.21.1";
+
+  src = fetchFromGitHub {
+    owner = "transcend-io";
+    repo = "terragrunt-atlantis-config";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-lh5c5+Lj2jb9BrM3piG9ERUjluiN0f/sPYp4OyKzYBw=";
+  };
+
+  nativeBuildInputs = [
+    versionCheckHook
+  ];
+
+  versionCheckProgramArg = "version";
+
+  env.CGO_ENABLED = 0;
+
+  vendorHash = "sha256-brTFOsO2tgWlli1z0W0DGh7114iSlTMHlSZWoFmGdAs=";
+
+  doCheck = false; # requires network access
+
+  ldflags = [
+    "-X main.VERSION=${finalAttrs.version}"
+  ];
+
+  doInstallCheck = true;
```

```suggestion

  nativeInstallCheckInputs = [ versionCheckHook ];
  versionCheckProgramArg = "version";
  doInstallCheck = true;
```
These should be together for readability